### DISCUSSION TITLE
WebGPU support

### DISCUSSION THREAD

**Discussion by llfab** [mono/SkiaSharp#2940] [Q&A]
Tried to create an understanding for how ready Skia is for WebGPU instead of WebGL. I understand that there seems an approach under dev with Graphite. Do you understand better what the current status and the roadmap are?

Why WebGPU? Not sure if I even need it. Just looking for the best option to have shared textures between our own 3D renderer and the surrounding 2D UI renderer. In AvaloniaUI we use our own Vulkan 3D renderer and Avalonia's GPU interop to share textures and basically have a fully working Avalonia control that presents our 3D rendered texture with no drawbacks. Looking for a similar thing when running in a browser...

**Comment by ghost** [comment]
Any updates on this?
